service/controller/clusterapi/v29/resource/tcnp/testdata/case-0-basic-test.golden

```diff
@@ -0,0 +1,245 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Tenant Cluster Node Pool Cloud Formation Stack.
```

Comment:
Here is the rendered golden file now. Looking at it there is more to fix but we should do that separately.